### PR TITLE
Template tags for translation in 'About' page

### PR DESCRIPTION
**Pull request by ukey54**
This PR replaces {% blocktrans } with {% trans } template tag on About page only.

### CODE REVIEW

File: public_website/templates/public_website/about.html [hunk 1]
```diff
@@ -13,61 +13,26 @@ <h2>{% trans '...and Sawaliram was born!' %}</h2>
...
-            gathering in the village of Rasulia, in the Hoshangabad District of Madhya Pradesh State. 
-            They wished to develop an active, inquiring and experiment-based science program, a curriculum that 
-            was rooted in rural India.Their efforts led to the landmark 'Hoshangabad Science Teaching 
-            Program <a href="https://www.eklavya.in/past-work-top/programmes-past-top/hstp">(HSTP)</a>', one that was implemented in the next 30 years over large tracts of the State.' 
-         {% endblocktrans %}
+    {% trans "It was 1972 and a group of scientists, university academics, graduate students and school teachers were gathering in the village of Rasulia, in the Hoshangabad District of Madhya Pradesh State. They wished to develop an active, inquiring and experiment-based science program, a curriculum that was rooted in rural India.Their efforts led to the landmark 'Hoshangabad Science Teaching Program" %}<a href="https://www.eklavya.in/past-work-top/programmes-past-top/hstp">(HSTP)</a>'{%trans ", one that was implemented in the next 30 years over large tracts of the State." %}
```
Comment (subindevs) [inline public_website/templates/public_website/about.html:16]: can you please try writing the paragraph writing in multiple lines as in before

**Review by subindevs — CHANGES_REQUESTED** [review]

**Review by abhiprojectz — COMMENTED** [review]
Yes, Please split all sentences in `<p>` tag into separate trans.

File: public_website/templates/public_website/about.html (hunk 1 above)
Comment (ukey54) [reply public_website/templates/public_website/about.html]: How about now ?

**Comment by ukey54** [comment]
> Yes, Please split all sentences in `<p>` tag into separate trans.

Is it okay now ?

**Comment by abhiprojectz** [comment]
Just revert your current changes and don't include `<a>` tag in trans block.

**Comment by ukey54** [comment]
Closing PR because of trans block issue.

**Pull request closed without merging**